### PR TITLE
[READY] Replaces mining points with money, adjusts gulag payout

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
## About The Pull Request

Pretty much what it says on the tin.

## Why It's Good For The Game

Mining points are silly pre-cash from pre-economy times, we have actual money now, making use of that makes sense.

If some of those items are "too powerful to be allowed on the station", then the item, its cost or the amount stocked should be adjusted. 

The gulag change serves to get gulaged players back onto the station, by:
1. giving security players a reason to not strip-gulag people (their payout depends on this)
2. giving the gulaged player a reason to get back (their ID and stuff might actually be reclaimable), and they get a small payout

It also pre-emptively removes a money multiplier that we will want to remove if economy is to be a viable system.

## Changelog
:cl: Naksu
del: Removed mining points. Miners now earn cold hard cash in return for their efforts. Their regular paycheck has been reduced to compensate for this extra income.
change: ORM laser upgrades now increase the yield rather than the amount of mining points.
add: Golems' mining IDs are now linked to the Liberator's account.
tweak: Half of the gulag points earned go to the prisoner as cash and the other half is put into the security budget, unless the prisoner's real ID is not in the gulag reclamation console in which case the full sum is awarded to the prisoner.

/:cl:

TODO:
- [x] Give golem IDs a bank account untouchable by station crap
- [x] Remove ORM mining points per ore upgrades

### CODE REVIEW

**Comment by Barhandar** [comment]
economy was a mistake

**Comment by Whoneedspacee** [comment]
survival medipen meta starts today boys

**Comment by Eaglendia** [comment]
Wouldn't this double the actual value of all minerals?

**Comment by ExcessiveUseOfCobblestone** [comment]
We need to first fix the fact that lavaland and consequently the rewards/items from it are based on playing in this private magic land and would actually be troublesome to have widely available to the station.

**Comment by Arathianen** [comment]
I think lavaland is just content oriented to PvE co-op players...which is not a bad thing!

Escalation of power level is a feature across all jobs. Miners just don't really get that power directly from the station.

**Comment by Whoneedspacee** [comment]
every job gets op unique shit from playing their job, not so sure what’s different about minings magical world than hunkering down in robotics or xenobio.

**Comment by nemvar** [comment]
If we are removing mining points, we should just straight up remove the vendor with them and make miners order their gear from cargo.

**Comment by kriskog** [comment]
Dropping miners to assistant level paycheck is a bit harsh.

**Comment by wesoda25** [comment]
remove lavaland

**Comment by ArcaneMusic** [comment]
You're also forgetting that the miners... already have a way to directly convert their mining points to cash!
That's right, actually, there's still two. Drop your ores off at mining, get your mining points, and go and get all of your good ores like a stack of 50 plasma or something, and head to mining. Now, you can spend about 500 points on raw cash to start with, and while you're scrolling through and dropping 4000 mining points on money, the coin press will by then print your plasma into some 2000 extra cash in raw coins. 

There's... a lot that needs to be discussed on how exactly to tailor the mining economy into the cash economy to be sure. Either way, I think this isn't going to work.

**Comment by HeartwawyHorror** [comment]
Cue miners murdering heads of staff and the QM like street thugs looking for cash.

**Comment by Tlaltecuhtli** [comment]
medipen should cost way more like 1k at least, hardsuit 4k, the rest is joke/fluff so its fine

**Comment by actioninja** [comment]
mining being the dumb land of snowflake garbage design rears it's ugly head again.
remove mining.

**Comment by zxaber** [comment]
I can't wait for survival pens to be nerfed because now they're available to everyone.

**Comment by Whoneedspacee** [comment]
>someone makes bad pr for mining

>haha fuck mining (people who dislike mining)

>no dont do this (people who like mining)

>pr merged anyways

can we generalize this more folks take it away

**Comment by FantasticFwoosh** [comment]
I don't otherwise care about the other changes, but perma prisoners are there for a punishment to deter future bad behaviour, given money = power in terms of gathering commercially availible supplies. You can even get a set of premium security clothes from a hacked security vendor for 100's & 1000's of credits with no pre-required access.

If people want to earn money from mining as non-miners they should use the auxillary base with a self built ORM.

**Comment by ExcessiveUseOfCobblestone** [comment]
> every job gets op unique shit from playing their job, not so sure what’s different about minings magical world than hunkering down in robotics or xenobio.

They're not (in that regard) and both should be looked at rather than using one to excuse the other.

**Comment by Whoneedspacee** [comment]
you act like there’s a problem with that though, that is literally ss13. the game is fun and people like doing that.

**Comment by necromanceranne** [comment]
Presently, converting mining points into raw cash can net you lovely things like purchasing combat shotguns directly so you can go validhunt with your lavaland loot and without armory access. Or kill people in general. Or give a gift to the QM he will actually appreciate since you likely have an obscene amount of those points and giving everyone a hardsuit will quickly grow kinda stale, but being the sole reason cargo can throw an endless pizza party is the gift that keeps on giving.

Basically what I'm saying is that this should be first be test merged because it will likely get really fucking silly with the cargo budget being what it is and miners and techs both wanting gear round start.

**Comment by ExcessiveUseOfCobblestone** [comment]
> you act like there’s a problem with that though, that is literally ss13. the game is fun and people like doing that.

of course people enjoyed overpowered gear, and yes it might be fun in the same way San Andreas cheat codes is a blast.

That however does not make it good for the game.

**Comment by anconfuzedrock** [comment]
I think the main difference between mining and other departments is that with fully upgraded engineering you can create forcefields and block gas flow while with 10 minutes as a miner you get a wizard sword and an 18tc stand

**Comment by Whoneedspacee** [comment]
fully upgraded engineering can bomb the entire station to nothing, create radiation hell, generate massive amounts of science points, do fusion for stimulum which literally stops all stuns and speeds you up, create flamethrowers that while useless to fire resistant people can literally instantly fuck up anyone without that.

all departments have practically antag only stuff, mining is no different. it's not like i can just run around using the staff of storms or something similar yknow.

**Comment by FloranOtten** [comment]
> > every job gets op unique shit from playing their job, not so sure what’s different about minings magical world than hunkering down in robotics or xenobio.
> 
> They're not (in that regard) and both should be looked at rather than using one to excuse the other.


You forgot Cargo, which gets money and thus can order a fuckload of things

You forgot RnD which, by the end of the shift, can create holes in spacetime, several weapons, can desync you from time and space, can bomb you to smithereens with maxcaps etc.

You forgot Engi, which can, with minor edits, create an infinite loop that spreads lethal rads all over

You forgot Atmospherics, which can turn the entire station into an oven so hot even their own hardsuit fails

You forgot Botany, which can create spiders, esword level plants, earthsblood, kudzu, melons that slip you and turn you into a catgirl, etc.

Chemistry, which can get a lot of fancy chemicals and, with some effort, even create maxcaps of their own.

There's more I can't think of, but yeah. It'd be quicker listing departments that don't get stronger as the round progresses (Sec, Medbay and Service. I say these three because they don't get stronger under their own power. All of them benefit from the advances for RnD, engi and botany)

**Comment by optimumtact** [comment]
Just because we've crashed the car before doesn't mean we should keep crashing it

**Comment by adamkad1** [comment]
Its time for profitcurity! arrest random assistants and send them to gulag!

**Review by AnturK — DISMISSED** [review]
Looks good, needs tgui update.

**Review by ninjanomnom — APPROVED** [review]

**Comment by Whoneedspacee** [comment]
i cannot fathom how bad of an idea this was but meh more powergame for me

**Comment by adamkad1** [comment]
yeah, ignore my suggestions

**Comment by MemedHams** [comment]
So i have several issues with this.
1: This makes mining equipment available to anyone who is capable of making a Mining rewards circuit, or accessing the one in mining.
2: The total nerf of useful mining equipment, in order to prevent people with budget cards/people with money from powergaming, seems more like a punishment to mining that doesn't really punish anyone
3: it will be far more difficult for cargo to do their job now overall, since the entire crew has an active interest in retaining their own money and sellables, and to illegally/legally extract wealth from cargo. I can see people breaking into cargo to fufill bounties and take the money with this.
Overall this creates many issues, only to solve what is really a minor nitpick on a beneficial mechanic which encourages that miners do their bloody job.
**This has been the worst trade deal in the history of trade deals, maybe ever.**
In all seriousness, this is certainly a head-scratching PR, and it's even more baffling that **this was merged**. If this isn't a symptom of a far greater issue i'm not sure what is.

**Comment by adamkad1** [comment]
> 3: it will be far more difficult for cargo to do their job now overall, since the entire crew has an active interest in retaining their own money and sellables, and to illegally/legally extract wealth from cargo. I can see people breaking into cargo to fulfill bounties and take the money with this.
 
Its not like they didnt before? Its just that you can get some cool stuff but most of the stuff you could order substitues in cargo, so its no big deal

**Comment by Cyprex** [comment]
If you were going to do this you really should have made it a proper vendor, requiring ID insertion is just very annoying at this point.
